functions.js

```diff
@@ -0,0 +1,260 @@
+const btn = document.querySelector('.humb');
+const navLink = document.querySelector('#app-header-menu');
+const closeButton = document.querySelector('#humb-cancelor');
+const toclose = document.querySelectorAll('.nav-link');
+const menuheader = document.querySelector('#app-header-info');
+
+// To open the menu bar
+function displays() {
+  navLink.classList.add('app-header-menu-visible');
+  const menu = document.querySelector('#welcome');
+  menu.classList.add('welcome-invisible');
+  btn.classList.add('humb-invisible');
+  closeButton.classList.add('humb-cancel-visible');
+  menuheader.classList.add('app-header-hide');
+}
+
+// To close the menu bar
+function close() {
+  navLink.classList.remove('app-header-menu-visible');
+  const menu = document.querySelector('#welcome');
+  menu.classList.remove('welcome-invisible');
+  btn.classList.remove('humb-invisible');
+  closeButton.classList.remove('humb-cancel-visible');
+  menuheader.classList.remove('app-header-hide');
+}
+
+// Adding event Listener humburger menu and the close icon
+btn.addEventListener('click', display)
```

Comment:
A semicolon is missing at the end of the statement. You might consider adding a trailing semicolon to fix this.